src/main/java/org/jala/efeeder/suggestion/SetWinnerPlaceCommand.java

```diff
@@ -54,12 +56,20 @@ public Out execute(In parameters) throws Exception {
 		}
 
 		try {
-			String roomId = "vote_" + Integer.toString(feastId);
+			String roomId = WebsocketsConstants.voteRoomPrefix + Integer.toString(feastId);
+			String homeRoomId = WebsocketsConstants.homeRoom;
 			List<MessageEvent> events = new ArrayList<>();
 			events.add(MessageEvent.newBuilder().setEvent(
-					CloseVotingEvent.newBuilder().build()).build());
-			CommandEndpoint.getRoomManager().getRoom(roomId).sendMessage(
-					MessageContext.newBuilder().setUser(0).setRoom(roomId).setEvents(events).build());
+					ChangeFoodMeetingStatusEvent.newBuilder()
+						.setIdFoodMeeting(feastId)
+						.setIdUser(parameters.getUser().getId())
+						.setNewStatus(FoodMeetingStatus.Order.name())
+						.build())
+					.build());
+			
```

Comment:
I Think this space helps to separate the events creation from the message creation, and helps for readability